src/components/chart/index.js

```diff
@@ -1,7 +1,7 @@
-import Api from '../../api'
+import Api from '../../api/index.js'
```

Comment:
You should not need that. Also you should not ever need or use `.js` extension in imports.